provisioner/deployment/docker/Dockerfile

```diff
@@ -19,9 +19,14 @@ ADD . .
 RUN cd provisioner; \
 	CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' -o local-volume-provisioner ./cmd
 
+# For security, we use kubernetes community maintained debian base image.
+# https://github.com/kubernetes/kubernetes/blob/master/build/debian-base/
 FROM k8s.gcr.io/debian-base-amd64:0.4.0
 
-RUN clean-install \
+# Keep packages up to date and install packages for our needs.
+RUN apt-get update \
```

Comment:
is it possible to only update the packages when running CI builds vs doing a local build?  This may increase the container build time for local dev/testing

Reply:
docker build has builtin cache mechanism with which docker will reuse cache if nothing change in build context. In local dev/testing, docker will run all RUN commands to build image for the first time, and will only build local-volume-provisioner when user made some change in repo.